azure-storage-blob/src/Blob/BlobRestProxy.php

```diff
@@ -379,8 +379,19 @@ private function createPath($container, $blob = '')
     private function getBlobUrl($container, $blob)
     {
         $encodedBlob = $this->createPath($container, $blob);
+        $uri = $this->getPsrPrimaryUri();
+        $exPath = $uri->getPath();
 
-        return (string)($this->getPsrPrimaryUri()->withPath($encodedBlob));
+        if ($exPath != '') {
+            //Remove the duplicated slash in the path.
+            if ($encodedBlob[0] == '/') {
+                $encodedBlob = $exPath . substr($encodedBlob, 1);
```

Comment:
Thanks for your contribution!!

Will last character of ```$exPath``` always be a ```/``` ? I'm worried about it which may have future changes.
May be we can just join them and string replace any double ```//``` into single ```/```.

Reply:
You're rigth, we should be sure that the last char of the `$exPath` is a `/` to later join it with the `$encodedBlob` path. I'll have a look at it